tests/unit/parser.js

```diff
@@ -213,6 +213,47 @@ exports.options = function (test) {
   test.done();
 };
 
+exports.emptyDirectives = function (test) {
+  TestRun(test)
+    .addError(1, "Bad option value.")
+    .test('/* global */');
+
+  TestRun(test)
+    .addError(1, "Bad option value.")
+    .test('/* global : */');
+
+  TestRun(test)
+    .addError(1, "Bad option value.")
+    .test('/* global -: */');
+
+  TestRun(test)
+    .test('/* global foo, bar, baz, */');
```

Comment:
Sure: I just pushed a commit to address this https://github.com/jshint/jshint/commit/8e4307d9571964fc65a3ee381c7b6d57f72b3dee . Is that what you had in mind?